srml/session/src/lib.rs

```diff
@@ -356,9 +358,40 @@ mod tests {
 	pub struct Test;
 	impl consensus::Trait for Test {
 		type Log = DigestItem;
-		type SessionKey = UintAuthorityId;
+		type SessionKey = TestSessionKey;
 		type InherentOfflineReport = ();
 	}
+
+	#[derive(Clone, Debug, Default, PartialEq, Eq, Encode, Decode, Serialize, Deserialize)]
+	pub struct TestSessionKey(u64, u64);
+
+	impl Into<sr25519::Public> for TestSessionKey {
```

Comment:
`From<TestSessionKey> for sr25519::Public` should be preferred over `Into` (you will get `Into` for free anyway)